docs/commands.md

```diff
@@ -340,7 +340,7 @@ TS.INFO key
 TS.INFO temperature:2:32
  1) lastTimestamp
  2) (integer) 1548149279
- 3) retentionSecs
+ 3) retentionTime
```

Comment:
@gkorland just checking here if this is not a replace all but effectively in the code

Reply:
both :) replace all and effectively in the code